nibabel/processing.py

```diff
@@ -353,6 +354,10 @@ def conform(
     order : int, optional
         The order of the spline interpolation, default is 3.  The order has to
         be in the range 0-5 (see ``scipy.ndimage.affine_transform``)
+    mode : str, optional
+        Points outside the boundaries of the input are filled according to the
+        given mode ('constant', 'nearest', 'reflect' or 'wrap'). Default is
+        'constant' (see scipy.ndimage.affine_transform)
```

Comment:
We should be able to link this.

```suggestion
        'constant' (see :func:`scipy.ndimage.affine_transform`)
```